react/Rating/Rating.js

```diff
@@ -61,7 +62,9 @@ const Rating = ({
           );
         })}
         {showTextRating &&
-          <span className={styles.textRating}>{rating.toFixed(1)}</span>
+          <span className={styles.textRating}>
```

Comment:
I don't think it will be obvious that `textRating` is required in order to have the comment show up. We probably should avoid having a dependency here.